pipeline-maven-api/src/main/java/org/jenkinsci/plugins/pipeline/maven/dao/PipelineMavenPluginNullDao.java

```diff
@@ -40,12 +43,34 @@
 /**
  * @author <a href="mailto:cleclerc@cloudbees.com">Cyrille Le Clerc</a>
  */
+@Extension
 public class PipelineMavenPluginNullDao implements PipelineMavenPluginDao {
     private static Logger LOGGER = Logger.getLogger(PipelineMavenPluginNullDao.class.getName());
 
+    @Override
+    public String getDescription() {
+        // TODO i18n
+        return "Pipeline Maven Plugin no storage mode";
```

Comment:
one i18n missed ;)

Reply:
well this one is an interesting one.... Not sure if you have noticed but I tried to keep backward compact for package name and classes name even if changing maven module/jenkins plugin.
So here if we add a Messages class in this package there will be conflict with the other one.
I wonder if we should change package at least for the database module?
wdyt?

Reply:
all fixed. It was a real mistake to have same package with classes in different maven module/jenkins plugin

Reply:
Many many thanks !